proto/ibc/core/channel/v1/tx.proto

```diff
@@ -294,6 +294,16 @@ message MsgChannelUpgradeAckResponse {}
 message MsgChannelUpgradeConfirm {
   option (gogoproto.equal)           = false;
   option (gogoproto.goproto_getters) = false;
+  string  port_id                    = 1 [(gogoproto.moretags) = "yaml:\"port_id\""];
+  string  channel_id                 = 2 [(gogoproto.moretags) = "yaml:\"channel_id\""];
+  Channel counterparty_channel       = 3
+      [(gogoproto.moretags) = "yaml:\"counterparty_channel\"", (gogoproto.nullable) = false];
+  bytes                     proof_channel          = 4 [(gogoproto.moretags) = "yaml:\"proof_channel\""];
+  bytes                     proof_upgrade_error    = 5 [(gogoproto.moretags) = "yaml:\"proof_upgrade_error\""];
```

Comment:
I'm not certain about the type required here.

This was based on the following values from the spec

```js
    proofChannel: CommitmentProof,
    proofUpgradeError: CommitmentProof,
    proofUpgradeSequence: CommitmentProof,
```

Reply:
i believe you are correct that it is bytes, since that's how we pass other proofs like that required for ie: [initializing a channOpen ](https://github.com/cosmos/ibc-go/blob/6b4aa9a50096d69f1e392e25809f0efc15b3a5f4/proto/ibc/core/channel/v1/tx.proto#L87)